Shared/Samples/Edit and sync features with feature service/EditAndSyncFeaturesWithFeatureServiceView.swift

```diff
@@ -0,0 +1,193 @@
+                                        right: -geometryProxy.safeAreaInsets.trailing + 20
+                                    )
+                                )
+                                guard let extent = mapViewProxy.envelope(
+                                    fromViewRect: viewRect
+                                ) else { return }
+                                
+                                // Generates the geodatabase using the envelope.
+                                try await model.generateGeodatabase(extent: extent)
+                                statusText = action.completionMessage
+                            case .selectFeature(let screenPoint):
+                                // Identifies and selects a feature at the tapped screen point.
+                                let identifyLayerResults = try await mapViewProxy.identifyLayers(
+                                    screenPoint: screenPoint,
+                                    tolerance: 22,
+                                    maximumResultsPerLayer: 1
+                                )
+                                
+                                model.selectFeature(identifyLayerResults: identifyLayerResults)
+                                if model.selectedFeature != nil {
+                                    statusText = action.completionMessage
+                                }
+                            case .moveSelectedFeature(mapPoint: let mapPoint):
+                                try await model.moveSelectedFeature(point: mapPoint)
+                                statusText = action.completionMessage
+                            case .sync:
+                                try await model.syncGeodatabase()
+                                statusText = action.completionMessage
+                            case .cancelJob:
+                                await model.cancelJob()
+                                statusText = action.completionMessage
+                            case .reset:
+                                await model.reset()
+                                await mapViewProxy.setViewpoint(model.map.initialViewpoint!)
+                                selectedAction = .setUpMap
+                                return
+                            }
+                        } catch {
+                            self.error = error
+                        }
+                        
+                        selectedAction = nil
+                    }
+                    .errorAlert(presentingError: $error)
+            }
+        }
+        .overlay(alignment: .top) {
+            VStack {
+                Text(statusText)
+                    .multilineTextAlignment(.center)
+                    .frame(maxWidth: .infinity, alignment: .center)
+                    .padding(8)
+                    .background(.regularMaterial, ignoresSafeAreaEdges: .horizontal)
+                
+                Rectangle()
```

Comment:
It might be worth a comment "Drawing red rectangle representing offline map area"